runtime/src/main/java/org/corfudb/protocols/logprotocol/MultiObjectSMREntry.java

```diff
@@ -93,39 +103,115 @@ public void mergeInto(MultiObjectSMREntry other) {
      */
     @Override
     public void deserializeBuffer(ByteBuf b, CorfuRuntime rt) {
-        super.deserializeBuffer(b, rt);
+        Optional<Timer> deserializeTimer = MeterRegistryProvider.getInstance()
```

Comment:
Looks like this part will only deserialize the ByteBuffer to a mapping of <StreamID, byte array>, do we also need the part that deserialize the byte array to the actual MultiSMREntry?

Reply:
Oh you mean - the time it takes to do the lazy deserialization? I can include that